src/main/java/mate/academy/spring/model/dto/response/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy.spring.model.dto.response;
+
+import java.util.List;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ShoppingCartResponseDto {
+    private Long id;
+    private List<Long> ticketsId;
+
```

Comment:
```suggestion
      private Long userId;
```
not sure that without this field you can get userId in mapper